pkg/http/api_test.go

```diff
@@ -820,12 +820,27 @@ func TestGetStatusCodeFromMetadata(t *testing.T) {
 }
 
 func TestGetMetadataFromRequest(t *testing.T) {
+	// set
+	r, _ := gohttp.NewRequest(gohttp.MethodGet, "http://test.example.com/resource?metadata.test=test&&other=other", nil)
```

Comment:
Uhm I just copied this from the fasthttp test below which did have `&&`. I assume it's intended for testing purposes. Just like `other` is included because it should not be returned.